[SF 5.3] Impossible to make functionnal test with SQLite: Operation 'AbstractPlatform::getListDatabasesSQL' is not supported by platform.

### Preconditions
<!--- Provide a more detailed information of environment you use -->
<!--- Bundle version, tag, HEAD, etc., PHP version, etc.. -->
My dependencies on a fresh Symfony app are:
```json
"require": {
    "php": ">=7.4",
    "ext-ctype": "*",
    "ext-iconv": "*",
    "composer/package-versions-deprecated": "1.11.99.4",
    "doctrine/doctrine-bundle": "^2.4",
    "doctrine/doctrine-migrations-bundle": "^3.2",
    "doctrine/orm": "^2.10",
    "symfony/console": "5.3.*",
    "symfony/dotenv": "5.3.*",
    "symfony/flex": "^1.3.1",
    "symfony/framework-bundle": "5.3.*",
    "symfony/proxy-manager-bridge": "5.3.*",
    "symfony/runtime": "5.3.*",
    "symfony/yaml": "5.3.*"
},
"require-dev": {
    "dama/doctrine-test-bundle": "^6.6",
    "doctrine/doctrine-fixtures-bundle": "^3.4",
    "liip/test-fixtures-bundle": "^2.0.0",
    "phpunit/phpunit": "^9.5",
    "symfony/browser-kit": "5.3.*",
    "symfony/css-selector": "5.3.*",
    "symfony/maker-bundle": "^1.34",
    "symfony/phpunit-bridge": "^5.3"
},
```
1. PHP 7.4 & PHP 8
2. Symfony 5.3

### Steps to reproduce
<!--- Provide a set of unambiguous steps to reproduce this bug include code, if relevant  -->
1. Install a fresh symfony application
```
symfony new app
cd app
composer require --dev phpunit/phpunit symfony/test-pack
```
2. Create a WebTestCase in `app/tests/DummyFunctionnalTest.php`
3. Install Doctrine and the maker bundle
```
composer require symfony/orm-pack
composer require --dev symfony/maker-bundle
```
4. Set the `DATABASE_URL` in the `.env.test` file :
```
DATABASE_URL="sqlite:///%kernel.project_dir%/var/app.db"
```
5. Create an entity with the maker
6. Create a fixture that creates an entity
```php
class PostFixtures extends Fixture
{
    public function load(ObjectManager $manager): void
    {
        $post = new Post();

        $post->setTitle("Titre");
        $post->setContent("Contenu");
        $post->setCreatedAt(new DateTimeImmutable());

        $manager->persist($post);
        $manager->flush();
    }
}
```
7. Install LiipTestFixturesBundle: 
```
composer require --dev liip/test-fixtures-bundle:^2.0.0
```
8. Write a test :
```php
class DummyFunctionalTest extends WebTestCase
{
    protected KernelBrowser $client;
    private AbstractDatabaseTool $databaseTool;

    protected function setUp(): void
    {
        parent::setUp();

        $this->client = static::createClient();
        $this->databaseTool = $this->client->getContainer()->get(DatabaseToolCollection::class)->get();
    }

    public function testDummy(): void
    {
        $this->databaseTool->loadFixtures([PostFixtures::class]);

        $this->assertNotNull($this->client->getContainer()->get(PostRepository::class)->findAll());
    }
}
```
9. Run the tests

### Expected result
<!--- Tell us what should happen -->
1. I expect the test to run and reach the assert statement

### Actual result
I got this error when I run tests:
```
./bin/phpunit
PHPUnit 9.5.10 by Sebastian Bergmann and contributors.

Testing 
E                                                                   1 / 1 (100%)

Time: 00:00.100, Memory: 16.00 MB

There was 1 error:

1) App\Tests\DummyIntegrationTest::testDummy
Doctrine\DBAL\Exception: Operation 'Doctrine\DBAL\Platforms\AbstractPlatform::getListDatabasesSQL' is not supported by platform.

/app/vendor/doctrine/dbal/src/Exception.php:22
/app/vendor/doctrine/dbal/src/Platforms/AbstractPlatform.php:2749
/app/vendor/doctrine/dbal/src/Schema/AbstractSchemaManager.php:102
/app/vendor/liip/test-fixtures-bundle/src/Services/DatabaseTools/ORMDatabaseTool.php:163
/app/vendor/liip/test-fixtures-bundle/src/Services/DatabaseTools/ORMDatabaseTool.php:60
/app/tests/DummyFunctionalTest.php:27

ERRORS!
Tests: 1, Assertions: 0, Errors: 1.
```

--- a/src/Services/DatabaseToolCollection.php
+++ b/src/Services/DatabaseToolCollection.php
@@ -47,7 +47,7 @@ final class DatabaseToolCollection
     {
         /** @var ManagerRegistry $registry */
         $registry = $this->container->get($registryName);
-        $driverName = ('ORM' === $registry->getName()) ? \get_class($registry->getConnection()->getDriver()) : 'default';
+        $driverName = ('ORM' === $registry->getName()) ? \get_class($registry->getConnection()->getDatabasePlatform()) : 'default';
 
         $databaseTool = isset($this->items[$registry->getName()][$driverName])
             ? $this->items[$registry->getName()][$driverName]

--- a/src/Services/DatabaseTools/ORMSqliteDatabaseTool.php
+++ b/src/Services/DatabaseTools/ORMSqliteDatabaseTool.php
@@ -36,7 +36,7 @@ class ORMSqliteDatabaseTool extends ORMDatabaseTool
 
     public function getDriverName(): string
     {
-        return Driver::class;
+        return SqlitePlatform::class;
     }
 
     public function loadFixtures(array $classNames = [], bool $append = false): AbstractExecutor

--- a/src/Services/DatabaseTools/ORMSqliteDatabaseTool.php
+++ b/src/Services/DatabaseTools/ORMSqliteDatabaseTool.php
@@ -15,7 +15,6 @@ namespace Liip\TestFixturesBundle\Services\DatabaseTools;
 
 use Doctrine\Common\DataFixtures\Executor\AbstractExecutor;
 use Doctrine\Common\DataFixtures\ProxyReferenceRepository;
-use Doctrine\DBAL\Driver\PDO\SQLite\Driver;
 use Doctrine\DBAL\Platforms\SqlitePlatform;
 use Doctrine\ORM\Tools\SchemaTool;
 use Liip\TestFixturesBundle\Event\FixtureEvent;

